### ISSUE TITLE
textSize is variableient

### ISSUE THREAD

**Issue by teamhide** [SeregPie/THREE.TextSprite#11]
![1](https://user-images.githubusercontent.com/8403122/45661201-58127480-bb37-11e8-9e18-91da6c652900.gif)

Refer this picture, font-size is variableient when camera zoom in/out.

But I fixed font-size like this.

```
  for ( let i = 0; i < 100; i ++ ) {
     let sprite = new THREE.TextSprite({
      textSize: 2,
      redrawInterval: 1,
      texture: {
        text: 'TEST TEST',
        fontFamily: 'Arial, Helvetica, sans-serif',
      },
      material: {
        color: 'white',
      },
    });
    sprite.position.x = Math.random() * 80-40;
    sprite.position.y = Math.random() * 80-40;
    sprite.position.z = Math.random() * 1000-40;
    scene.add(sprite);
  }
```

I defined `textSize:2`. Why textSize is variableient?

Is it a bug?

**Comment by SeregPie** [comment]
Can you provide the whole code? Upload it somewhere like codepen.

**Comment by teamhide** [comment]
I upload my code via codepen -> [https://codepen.io/anon/pen/RYeOxW](https://codepen.io/anon/pen/RYeOxW)

**Comment by teamhide** [comment]
Mouse Left Click: Move forward / Right Click: Move backward

**Comment by SeregPie** [comment]
It looks normal on my side. Nothing like your image.

Do you experience the same behavior in the demo?
https://seregpie.github.io/THREE.TextSprite/

What is your browser? Do you use windows, mac, android, iphone?

**Comment by teamhide** [comment]
No. Your example works perfectly.

I'm using Chrome browser and with iMac.

In your computer, works fine?

**Comment by teamhide** [comment]
I tested on iPhone(Safari), it works fine.

But Windows(Chrome) doesn't works too.

**Comment by teamhide** [comment]
I found that this bug occured when textSize is too small.

Likewise your example(Despite it's textSize is 10) has same situation.

Look at the text's size that I checked with red circle.

<img width="311" alt="2018-09-18 5 02 47" src="https://user-images.githubusercontent.com/8403122/45673088-c0c41600-bb64-11e8-8f38-51eba3504d5e.png">


![123](https://user-images.githubusercontent.com/8403122/45673024-996d4900-bb64-11e8-9317-8d4f2268edcb.gif)
